apps/predictions/lib/repo.ex

```diff
@@ -15,14 +15,29 @@ defmodule Predictions.Repo do
   def all(opts) when is_list(opts) and opts != [] do
     _ = Logger.info("predictions_repo_all_cache=call")
 
+    opts
+    |> add_all_optional_params()
+    |> cache(&fetch/1)
+    |> filter_by_min_time(Keyword.get(opts, :min_time))
+    |> load_from_other_repos
+  end
+
+  def all_no_cache(opts) when is_list(opts) and opts != [] do
```

Comment:
do you want a `filter_by_min_time` in here or no?

Reply:
no, this was intentionally omitted.  It is helpful in the version that was cached, but for the version that is non-cached it has not yet been necessary